pages/account/manage/links.js

```diff
@@ -45,6 +50,12 @@ export default function ManageLinks({ BASE_URL, username, links }) {
       />
 
       <Page>
+        {success ? (
```

Comment:
Ternary not needed, just a check if `true`, that way we can reduce the code required and keep it simple

```suggestion
        {success && (
```